src/charts/scatter-plot.js

```diff
@@ -870,7 +872,15 @@ define(function(require) {
             highlightCrossHairLabelsContainer.attr('opacity', opacityIndex);
         }
 
-        // API
+        const getXAxisFormat = () => {
```

Comment:
nit: let's keep the function declaration format.

Reply:
Yeah, documenting it with comments is preferred. You could do something like:

```
        /**
         * Sets x-axis format based on 'xAxisFormatType' setter
         * Applies `timeFormat` formatter function if custom 'xAxisFormatType' is provided
         * @return {Function}
         * @private
         */
```

Reply:
sure, no problem